src/rp2_common/pico_cyw43_arch/cyw43_arch_freertos.c

```diff
@@ -39,6 +39,10 @@
 
 #define CYW43_GPIO_IRQ_HANDLER_PRIORITY 0x40
 
+#ifndef CYW43_TASK_STACK_SIZE
+#define CYW43_TASK_STACK_SIZE configMINIMAL_STACK_SIZE
```

Comment:
yeah, i wonder if `configMINIMAL_STACK_SIZE` is ever a good value ;-) we should maybe set it to something reasonable that works for the variety of normal uses that suggested this issue

Reply:
(i.e. `configMINIMAL_STACK_SIZE` i think is intended for basically a task which hardly uses any stack at all)